pydecred/rpc.py

```diff
@@ -585,6 +734,236 @@ def get(k, obj):
     return obj[k] if k in obj else None
 
 
+class GetChainTipsResult:
+    def __init__(
+        self, height, blockHash, branchLen, status,
+    ):
+        """
+        Args:
+            height (int): The height of the chain tip.
+            blockHash (str): The block hash of the chain tip.
+            branchLen (int): The length of the branch that connects the tip to
+                the main chain (0 for the main chain tip).
+            status (str): The status of the chain (active, invalid, headers-only,
+                valid-fork, valid-headers).
+        """
+        self.height = height
+        self.blockHash = blockHash
+        self.branchLen = branchLen
+        self.status = status
+
+    @staticmethod
+    def parse(obj):
+        """
+        Parse the GetChainTipsResult from the decoded RPC response.
+
+        Args:
+            obj (object): The decoded dcrd RPC response.
+
+        Returns:
+            GetChainTipsResult: The GetChainTipsResult.
+        """
+        return GetChainTipsResult(
+            height=obj["height"],
+            blockHash=obj["hash"],
+            branchLen=obj["branchlen"],
+            status=obj["status"],
+        )
+
+
+class GetBlockSubsidyResult:
+    """getblocksubsidyresult"""
+
+    def __init__(
+        self, developer, subsidyPoS, subsidyPoW, total,
+    ):
+        """
+        Args:
+            developer (int): The developer subsidy.
+            subsidyPoS (int): The Proof-of-Stake subsidy.
+            subsidyPoW (int): The Proof-of-Work subsidy.
```

Comment:
Consider just `pos` and `pow`, since `subsidy` is already in the class name.

Reply:
pow is a builtin I think. https://docs.python.org/3/library/functions.html is it ok to use here?

Reply:
Oh yeah. Good to go then.

Reply:
I think it looks better now, since pow is the only field affected, does this work?